src/Coduo/PHPHumanizer/String/HtmlTruncate.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Coduo\PHPHumanizer\String;
+
+class HtmlTruncate extends TextTruncate
```

Comment:
The whole point of separating the interface was to not extend in HtmlTruncate from TextTruncate. Those 2 are standalone strategies of string truncating.